layouts/default/68_iso/readme.md

```diff
@@ -0,0 +1,5 @@
+# 68_iso
+
+    LAYOUT_68_iso
+
+This is the 68 (actually 69) key ISO layout made popular by boards such as the Magicforce 68 and Varmilo VA68M.
```

Comment:
If we want to get technical:

```suggestion
This is the 68 (actually 69) key ISO layout made popular by boards such as the Magicforce 69-key and Varmilo VA69M.
```

I don't actually know what the ISO Magicforce 68 is called officially, but Varmilo's board is the VA69M.